profiles/common/modules/features/nexteuropa_dgt_connector/ne_tmgmt_dgt_ftt_translator/ne_tmgmt_dgt_ftt_translator.module

```diff
@@ -132,16 +135,12 @@ function ne_tmgmt_dgt_ftt_translator_entity_property_info() {
  * Callback for requests/responses sent by the DGT Services (Poetry).
  */
 function _ne_tmgmt_dgt_ftt_translator_callback() {
-  // Log the event.
-  // @todo: Remove this log.
-  watchdog(
-    'ne_tmgmt_dgt_ftt_translator',
-    'The translator DGT FTT receives a request/response - callback.',
-    array(),
-    WATCHDOG_INFO
-  );
 
-  $poetry = new Poetry();
+  $poetry = new Poetry([
+    'notification.username' => 'username',
+    'notification.password' => 'password',
+    'logger' => new Psr3Watchdog(),
```

Comment:
@voidtek we need to change these too, username and password will need to come from configuration.

Reply:
It's done, in the other branch.